lib/settings.js

```diff
@@ -1,11 +1,12 @@
 var _ = require('lodash')
 
-var Settings = function (config) {
+var Settings = function (config, logs) {
   this.config = config
+  this.config.logpath = logs.logsPath()
```

Comment:
This will override the value for all instances of the config object, don't mutate it. Should be fine to just show the config value straight away?

Reply:
My bad, still pretty new to nodejs and how vars are scoped in this project.
The assumption was also that any surrounding code wouldn't care about the config being altered, since future `logs.logsPath()` calls would then return the same value it previously overrode the config with.